docs/01_introduction/README.md

```diff
@@ -143,60 +143,60 @@ Some commonly used optional easyconfig parameters include:
 * ``easyblock``, which specifies which (generic) easyblock should be used;
 * ``sources`` and ``source_urls``, which specify the list of source files and where to download them;
 * ``dependencies`` and ``builddependencies``, which specify (drum roll...) the list of (build) dependencies;
-* ``configopts``, ``buildopts``, and ``installopts``, which speficy options for the configuration/build/install commands, resp.;
+* ``configopts``, ``buildopts``, and ``installopts``, which specify options for the configuration/build/install commands, respectively;
+
+If these parameters are not provided, default values will be provided or extrapolated. For clarity, specifying at least
+which `èasyblock`` to use is a good idea.
```

Comment:
the '`è`' is wrong here, but I also would drop this last sentence, since we don't specify `easyblock` in easyconfigs where a software-specific easyblock is used.

Reply:
Ah, right, the implicit magic kicks in. I completely forgot.